spp_demo/models/individual.py

```diff
@@ -16,6 +16,30 @@ class G2PIndividual(models.Model):
     z_cst_indv_pregnancy_start = fields.Date(
         "Pregnancy start"
     )  # We set a date to be able to clean it later
-    z_cst_indv_lactation_start = fields.Date(
-        "Lactation start"
+    z_cst_indv_pregnancy_end = fields.Date(
+        "Pregnancy end"
     )  # We set a date to be able to clean it later
+
+    # Maternal and neonatal outcomes
+    z_cst_indv_pregnancy_maternal_outcome = fields.Selection(
+        [
+            ("normal", "Normal"),
+            ("preterm", "Preterm birth"),
+            ("stillbirth", "Stillbirth"),
+        ],
+        "Maternal outcome",
+    )
+    z_cst_indv_pregnancy_neonatal_outcome = fields.Selection(
+        [
+            ("normal", "Neonatal death"),
+            ("conanomalies", "Congenital anomalies"),
+            ("infections", "Neonatal infections"),
+            ("loweight", "Low birth weight"),
+            ("small", "Small for gestational age"),
+            ("respiratory", "Respiratory distress"),
+            ("thrive", "Failure to thrive"),
+            ("seizures", "Neonatal seizures"),
+            ("neurodevdelay", "Neurodevelopmental delay"),
+        ],
+        "Neonatal outcome",
+    )
```

Comment:
Should this be here or in an event data where the last value is linked ? So we do not loose history of every birth.

Reply:
While working on this, I was also thinking that the entire pregnancy implementation (start/end/outcomes) should be event data. Like a "pregnancy event." Then the indicators could use the currently active pregnancy event, as only one pregnancy can be ongoing at the same time.

Should we spend the time to convert this into an event for the demo application? I agree it is a step in the right direction in showing how such data should be implemented in OpenSPP.